basxconnect/core/models/persons.py

```diff
@@ -25,7 +26,10 @@ def not_deleted(self, *args, **kwargs):
 class Person(models.Model):
     deleted = models.BooleanField(_("Deleted"), blank=True, default=False)
     personnumber = models.CharField(
-        _("Person number"), max_length=255, unique=True, blank=True
+        _("Person number"),
+        max_length=255,
+        unique=True,
+        default=lambda: f"__placeholder__{random.randint(100000, 999999)}",
```

Comment:
UUID instead of randint would have been "better" in some sense but I think it is overkill, this should in normal use cases always get overwritten in the "save" method. Keeping the default "ugly" is good to detect in case it gets leaked somewhere.